src/containers/App/index.tsx

```diff
@@ -122,20 +126,29 @@ class Arronax extends React.Component<Props, States> {
     };
   }
 
-  componentDidMount = () => {
-    const { fetchItems, selectedEntity } = this.props;
+  componentDidMount() {
+    const {
+      fetchItems,
+      selectedEntity,
+      attributes,
+      fetchAttributes,
+    } = this.props;
     fetchItems(selectedEntity);
-  };
+    if (attributes.length === 0) {
+      fetchAttributes();
```

Comment:
It was used twice.
Please remove on `FilterPanel` component.

Reply:
I moved the attributes fetching into thunks, so it is no longer be required in the components